output/typescript/types.ts

```diff
@@ -10701,8 +10710,8 @@ export interface MlJob {
   finished_time?: integer
   groups?: string[]
   job_id: Id
-  job_type: string
-  job_version: VersionString
+  job_type?: string
```

Comment:
Similar to the other fields I've mentioned, `job_type` would not be set when putting a job, but would be present when getting it back.